src/tag/styled-components.js

```diff
@@ -24,7 +24,7 @@ function getAlpha(props: SharedPropsT, hovered: boolean = false) {
   if (hovered) {
     return '0.8';
   }
-  return '0.92';
+  return '0.6';
```

Comment:
The `light` tag seems to have a too dark background in the light theme with the current change. Maybe move this value to the theme as well?